Screens/GeneralInfo.js

```diff
@@ -112,3 +91,22 @@ export default class HomeScreen extends React.Component {
     );
   }
 }
+
+const mapStateToProps = state => {
+  return {
+    character: state.characters[0]
+  }
+}
+
+const mapDispatchToProps = dispatch => {
+  return {
+    levelUp: id => {
+      dispatch(levelUp(id))
+    }
+  }
+}
+
+export default connect(
+  mapStateToProps, 
+  mapDispatchToProps
+)(HomeScreen)
```

Comment:
Connectors are what allow the state in the reducer to be used by the components of the app. It binds the state to the props of the component, but only allows changes to be made via the dispatcher methods.